### ISSUE TITLE
Jetty 11 build still depends on apache-jstl

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#5992]
**Jetty version**
11.0.1-SNAPSHOT

**Description**

```
[INFO] Jetty :: GCloud .................................... SUCCESS [  0.072 s]
[INFO] Jetty :: GCloud :: Session Manager ................. SUCCESS [  0.961 s]
[INFO] Jetty :: Memcached ................................. SUCCESS [  0.100 s]
[INFO] Jetty :: Memcached :: Sessions ..................... SUCCESS [  0.583 s]
[INFO] Jetty Tests :: Sessions :: Parent .................. SUCCESS [  0.081 s]
[INFO] Jetty Tests :: Sessions :: Common .................. SUCCESS [  1.463 s]
[INFO] Jetty :: NoSQL Session Managers .................... SUCCESS [  0.409 s]
[INFO] Jetty :: Test Keystore ............................. SUCCESS [  0.278 s]
[INFO] Jetty :: Home Assembly ............................. SUCCESS [ 10.394 s]
[INFO] Demo :: Embedded Jetty ............................. FAILURE [  7.296 s]
[INFO] Jetty :: Ant Plugin ................................ SKIPPED
[INFO] Jetty :: Websocket :: Core :: Tests ................ SKIPPED
...(snip)...
[INFO] Jetty :: Documentation :: AsciiDoctor Extensions ... SKIPPED
[INFO] Jetty :: Documentation ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:30 min
[INFO] Finished at: 2021-02-19T13:11:23-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project demos-jetty-embedded: Could not resolve dependencies for project
 org.eclipse.jetty.demos:demos-jetty-embedded:jar:11.0.1-SNAPSHOT: Could not find artifact
 org.eclipse.jetty:apache-jstl:jar:11.0.1-SNAPSHOT in oss-snapshots
 (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
```

**Comment by janbartel** [comment]
Seems there are these references still to apache-jstl:

> ./javadoc/pom.xml:      <artifactId>apache-jstl</artifactId>
> ./demos/embedded/pom.xml:      <artifactId>apache-jstl</artifactId>
> ./jetty-jspc-maven-plugin/pom.xml:      <artifactId>apache-jstl</artifactId>
> ./jetty-maven-plugin/pom.xml:      <artifactId>apache-jstl</artifactId>
> ./jetty-bom/pom.xml:        <artifactId>apache-jstl</artifactId>

**Comment by joakime** [comment]
should they be glassfish-jstl now?

**Comment by janbartel** [comment]
@joakime I'm fixing it now, PR on the way. If the PR builds in CI with no errors, then just merge it in.

**Comment by janbartel** [comment]
Fixed via #5993
